README.md

```diff
@@ -68,53 +68,62 @@ You would need to install `apt-transport-https` that supports adding an `https`
 
 here are the steps to perform:
 
-```
-apt-get install -y apt-transport-https
-apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
-add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
-apt-get update
+```bash
+# apt-get install -y apt-transport-https
+# apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 10779AB4
+# add-apt-repository "deb https://raw.githubusercontent.com/nisabek/icinga2-slack-notifications/master/reprepro general main"
+# apt-get update
 ```
 
 You are now ready to install the plugin with 
 
-`apt-get install icinga2-slack-notifications`
+```
+# apt-get install icinga2-slack-notifications
+```
 
 This will create the plugin files in the correct `icinga2` conf directory. 
 
 ### Installation using git
 
-1. clone the repository under your Icinga2 `/etc/icinga2/conf.d` directory
+1. Clone the repository under `/opt` and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory
```

Comment:
does it matter where it's cloned in the beginning?...

Reply:
Cloning the entire `icinga2-slack-notifications` repository when you only really want `icinga2-slack-notifications/src/slack-notifications` seems counterproductive to me, yes, as you will have to do moving anyway and then you'll have to deal with deleting the rest of the repo files unless you want icinga to try parse than as configs.

Reply:
how about this formulation:
"Clone the repository and copy the relevant folder into your Icinga2 `/etc/icinga2/conf.d` directory" ?

Reply:
Sounds good, committed